src/Sulu/Bundle/SnippetBundle/Admin/SnippetAdmin.php

```diff
@@ -206,6 +209,20 @@ public function getSecurityContexts()
         return $contexts;
     }
 
+    private function hasSomeDefaultSnippetPermission(): bool
```

Comment:
This is inspired by PageAdmin: https://github.com/sulu/sulu/blob/8191cc53629000fd4e3f25df7593d6ba189090f7/src/Sulu/Bundle/PageBundle/Admin/PageAdmin.php#L520